docs/modeling/compound-models.rst

```diff
@@ -1,9 +1,9 @@
-.. _compound-models:
+.. .. _compound-models:
```

Comment:
Why not just delete the text instead of commenting it out?

Reply:
I agree, this page needs updating